plugins/woocommerce/src/Internal/Orders/OrderAttributionController.php

```diff
@@ -254,7 +269,7 @@ private function enqueue_scripts_and_styles() {
 				'session'       => $session_length,
 				'ajaxurl'       => admin_url( 'admin-ajax.php' ),
 				'prefix'        => $this->field_prefix,
-				'allowTracking' => $allow_tracking,
+				'allowTracking' => 'yes' === $allow_tracking ? true : false,
```

Comment:
The ternary isn't necessary here
```suggestion
				'allowTracking' => 'yes' === $allow_tracking,
```

Reply:
Removed in a289aa7013602cde94c88ffe4e1cd060edeca954